src/Raven.Server/Program.cs

```diff
@@ -178,6 +178,24 @@ public static int Main(string[] args)
                             Console.WriteLine("TIP: type 'help' to list the available commands.");
                             Console.ForegroundColor = prevColor;
 
+                            if (configuration.Storage.IgnoreInvalidJournalErrors == true)
+                            {
+                                var message =
+                                    $"Server is running in dangerous mode because {RavenConfiguration.GetKey(x => x.Storage.IgnoreInvalidJournalErrors)} was set. " +
+                                    "It means that storages of databases, indexes and system one will be loaded regardless missing or corrupted journal files which " +
+                                    "are mandatory to properly load the storage. " +
+                                    "This switch is meant to be use only for recovery purposes. Please make sure that you won't use it on regular basis. ";
+
+                                if (Logger.IsInfoEnabled)
```

Comment:
Should probably be operations

Reply:
Fixed in https://github.com/ravendb/ravendb/pull/8609/commits/f82c40d830e0afbdf48da42e2457ecdf2f0964ee